packages/unmock-core/src/service-loaders/index.ts

```diff
@@ -0,0 +1,24 @@
+import debug from "debug";
+import * as _ from "lodash";
+import { FsServiceDefLoader } from "../fs-service-def-loader";
+import { IServiceDef } from "../interfaces";
+import { resolveUnmockDirectories } from "../utils";
+
+const debugLog = debug("unmock:node");
+
+export interface IServiceDefLoader {
+  loadSync(): IServiceDef[];
+}
+
+const createFsServiceDefLoader = (
```

Comment:
Is this also intended as a future factory function?

Reply:
Yep, and to hide some of the directory resolution.